### ISSUE TITLE
Docs feedback: /configuration/integrations/_index.md

### ISSUE THREAD

**Issue by adeverteuil** [grafana/agent#1831]
This is the doc for `metrics_relabel_configs` in the Agent docs:
https://grafana.com/docs/agent/latest/configuration/integrations/
```
  # Relabel metrics coming from the integration, allowing to drop series
  # from the integration that you don't care about.
  metric_relabel_configs:
    [ - <relabel_config> ... ]
```

That's not true. Here's a recap of the relabel configs.

- `relabel_config`: dynamically rewrite the label set of a target before it gets scraped
  - https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
- `metric_relabel_config`: applied to samples as the last step before ingestion. One use for this is to exclude time series that are too expensive to ingest. **There is no TSDB in the Grafana Agent so this config has no effect at all!**
   - https://prometheus.io/docs/prometheus/latest/configuration/configuration/#metric_relabel_configs
- `write_relabel_config`: same as `metric_relabel_config`, but is configured inside a remote_write config. **This** is where you put your rules to drop samples before sending them to the remote storage.
   - https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write

**Comment by rfratto** [comment]
> There is no TSDB in the Grafana Agent so this config has no effect at all!

I think there's a misunderstanding here. `metric_relabel_configs` are applied to samples ast the last step before ingestion to the _WAL_, not permanent storage. This is true for both Grafana Agent and Prometheus, and so the documentation is correct.

**Comment by rfratto** [comment]
That's right, `prometheus_remote_storage_samples_dropped_total` is a remote_write specific metric. However, using `write_relabel_config` will slightly increase CPU usage since remote_write has more samples in the WAL to process.
